### ISSUE TITLE
Toast-Swift not working in Xcode 14.3

### ISSUE THREAD

**Issue by GJDK** [scalessec/Toast-Swift#201]
Hi Team, 

Toast-Swift not working in Xcode 14.3. You can try with a plain vanilla app. 

Am importing Toast-Swift using Cocoapods and the version I use is 5.0.1.

Kindly help on this and thanks in advance.

**Comment by Swathiga-proj** [comment]
I updated Xcode - 14.3 today and Toast-Swift not working. Please kindly fix this issue.

**Comment by chinaTLJ** [comment]
Hi, I updated Xcode to 14.3 and Toast-Swift not working. real device  and simulator .
File not found: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/arc/libarclite_iphonesimulator.a

Linker command failed with exit code 1 (use -v to see invocation)
Please fix it.

**Comment by app4g** [comment]
I'm on Xcde 14.3.1 and Toast-Swift is working well..

**Comment by D-Lav** [comment]
I had the same issue after upgrading to Xcode 14.3.1

I've found this [thread](https://developer.apple.com/forums/thread/728021) with one of the answers saying:

> Xcode 14 only supports building for a deployment target of iOS 11. libarclite was necessary for older OS versions, but is now obsolete. If you're encountering errors referring to this library, you should audit every target in your project for those that declare support for a minimum deployment target under iOS 11, and update them to at least iOS 11, or something more recent than that.
I updated POD minimum deploymnet to 11.0 and it worked for me.



So setting the iOS target version to 11 or higher should fix the issue. In my case I needed to add the following code to the end of my pod file:
```
post_install do |pi|
  pi.pods_project.targets.each do |t|
    t.build_configurations.each do |config|
      config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '11.0'
    end
  end
end
```

**Comment by scalessec** [comment]
Fixed in `5.1.0`.
